aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ systemd:
         - name: 40-etcd-cluster.conf
           contents: |
             [Service]
-            Environment="ETCD_IMAGE_TAG=v3.3.13"
+            Environment="ETCD_IMAGE_TAG=v3.3.15"
```

Comment:
Why this etcd version? I though 3.3.10 was tested by kubernetes upstream (see https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.15.md#unchanged). Am I missing something?

Reply:
If we want to address security issues, then I think we should also update etcd to v3.3.15, because it's compiled with Go 1.12.9.
For more details: https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.3.md#v3315-2019-08-19

Reply:
Oh, cool. Thanks! These kind of info is important, imho, in the commit message